packages/store-ui/src/atoms/Skeleton/Skeleton.tsx

```diff
@@ -0,0 +1,17 @@
+import React, { forwardRef } from 'react'
+
+export interface SkeletonProps {
+  /**
+   * ID to find this component in testing tools (e.g.: cypress, testing library, and jest).
+   */
+  testId?: string
+}
+
+const Skeleton = forwardRef<HTMLDivElement, SkeletonProps>(function Skeleton(
+  { testId = 'store-skeleton', ...props },
```

Comment:
You only have the testId props, maybe this isn't necessary.
But... If you want that your component receives all div props, you need to extend HTMLDivElement